vhdbuilder/packer/install-dependencies.sh

```diff
@@ -216,10 +216,6 @@ AZURE_NPM_VERSIONS="
 1.1.5
 1.1.4
 1.1.2
-1.1.0
-1.0.33
```

Comment:
Sometimes it makes sense to leave the last patch of the minor version 1.0.33 and remove 1.1.2 instead. not sure if that is the case here.

Reply:
1.0.33 was enabled not long time ago, do we know what versions are we using in prod (by all clusters?)

Reply:
looks like most of them are in 1.14.
https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-config/commit/ccc5f39d72d52e27a11fe7007380d69d0b6d60c9?refName=refs%2Fheads%2Fmaster&path=%2Ftoggles%2Fazure-npm-image-tag.yaml&_a=compare

Reply:
Eliminating 1.0.33 from VHD is not an issue. 1.0.33 is the version released on Feb this year. We have did lots of upgrade and bug fix for NPM since then.